garak/attempt.py

```diff
@@ -105,6 +105,24 @@ def as_dict(self) -> dict:
             "messages": self.messages,
         }
 
+    @classmethod
+    def from_dict(cls, dicti):
+        """Initializes an attempt object from dictionary"""
+        attempt_obj = cls()
```

Comment:
Does this skip the attempt constructor? Can we add an explicit type signature to signal what `cls` is expected to be?